easybuild/tools/options.py

```diff
@@ -1063,6 +1065,10 @@ def _postprocess_config(self):
         if self.options.inject_checksums:
             self.options.pre_create_installdir = False
 
+        # imply --force with --disable-generate-devel-module
+        if not self.options.generate_devel_module:
+            self.options.force = True
```

Comment:
@smoors This seems weird to me, I don't think `--disable-generate-devel-module` should always imply `--force`?

Reply:
yeah I was not sure about that, I'll remove it.